### ISSUE TITLE
Amyloid goiter

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10458]
**Preferred term label:**

**Synonyms**
thyroid gland amyloidosis

**Definition (free text, please give PubMed ID)**
PMID: 19168408 Amyloid goiter has to be suspected in all patients with a progressive, rapidly growing, bilateral thyroid enlargement and a concomitant history of chronic inflammatory processes. Moreover, this should be suspected in patients who are known to have disease predisposing to amyloid deposition.

PMID: 19318742 Amyloidosis can involve the thyroid as part of a systemic disease or as a localized primary amyloid tumor.
Amyloid goiter is extremely rare and presents more frequently as an autopsy finding than in surgical material.
The thyroid gland may be asymptomatically involved by amyloid substance in nearly 30% to 80% of patients with primary or secondary amyloidosis.

**Parent term (use hpo.jax.org/app)**
Goiter HP:0000853
Amyloidosis HP:0011034

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ALECT2 amyloidosis ORPHA:439224
AA amyloidosis ORPHA:85445
AH amyloidosis ORPHA:442582

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000815
- label: Amyloid goiter
- parents: [Goiter (HP:0000853)](https://hpo.jax.org/app/browse/term/HP:0000853) | [Amyloidosis (HP:0011034)](https://hpo.jax.org/app/browse/term/HP:0011034)
- pmid: PMID:32874558|PMID:19318742|PMID:19168408
- def: Amyloidosis is defined as the accumulation of amorphous, proteinaceous material in different parts of the body. Amyloid can be deposited in the thyroid gland in such quantities to cause a clinically apparent enlargement of the gland known as Amyloid goiter.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10458
- synonyms: Thyroid gland amyloidosis(NONE)
